operator/builtin/input/k8sevent/k8s_event.go

```diff
@@ -125,6 +128,36 @@ func listNamespaces(ctx context.Context, client corev1.CoreV1Interface) ([]strin
 	return namespaces, nil
 }
 
+// startFindingNamespaces creates a goroutine that looks for namespaces to watch
+func (k *K8sEvents) startFindingNamespaces(ctx context.Context, client corev1.CoreV1Interface) {
+	k.wg.Add(1)
+	go func() {
+		defer k.wg.Done()
+
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-time.After(k.discoveryInterval.Duration):
```

Comment:
This will currently not start discovering until we hit the interval for the first time. Could we modify this so that it runs the discovery once first? A delay of 1 minute will be pretty noticeable

Reply:
Yeah that makes sense. This section was written before the default was decided, so that consideration slipped my notice. I'll add in an initial discovery.